src/components/experiment-tracking/parallel-coordinates/parallel-coordinates.js

```diff
@@ -3,6 +3,9 @@ import classnames from 'classnames';
 import * as d3 from 'd3';
 import { HoverStateContext } from '../utils/hover-state-context';
 import { v4 as uuidv4 } from 'uuid';
+import { MetricsChartsTooltip, tooltipDefaultProps } from '../tooltip/tooltip';
+
+import { formatTimestamp } from '../../../utils/date-utils';
```

Comment:
Can we augment this util function to return a date in the format specified by the design? That format is this: `DD MMM YYYY` (e.g. `12 Jan 2022`).

Reply:
I was going to check with @comym actually that whether we dont want to show the time at all, just a date but will update to match the design for now